### ISSUE TITLE
Alternative navigation of priiloader menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [DacoTaco/priiloader#3]
```
What steps will reproduce the problem?
1.Have unsynced remotes
2.Turn on wii

What is the expected output? What do you see instead?
You can't navigate the menu, because you cannot sync remotes or navigate
with other buttons

What version of priiloader are you using? On what ios/system menu?
Issue is present regardless.

Do you have any IOS patches installed? (if so which ios and what patch?)
Issue is present regardless

Basically, if you can make it possible to navigate without GC controller,
e.g. use front buttons to navigate menu.

```

Original issue reported on code.google.com by `themanwithnoaddress@gmail.com` on 9 Nov 2009 at 7:05
